src/app/pages/ParaTimesPage/TransactionAmount/index.tsx

```diff
@@ -101,16 +97,34 @@ export const TransactionAmount = () => {
                 name="amount"
                 style={{ width: '100%' }}
                 required
-                validate={amount =>
-                  validator(amount, balance!)
-                    ? {
-                        message: t('errors.insufficientBalance', 'Insufficient balance'),
-                        status: 'error',
-                      }
-                    : undefined
-                }
+                validate={[
+                  (amount: string) =>
+                    !new RegExp(`^\\d*(?:[.][0-9]{0,${decimals}})?$`).test(amount)
+                      ? {
+                          message: t(
+                            'paraTimes.validation.invalidDecimalValue',
+                            'Maximum of {{token}} decimal places is allowed',
```

Comment:
Minor inaccuracy that we probably don't care about, and don't want an uglier regex for:
0.400000000000000000000000000 shows "Maximum of 9 decimal places is allowed"